ghost/i18n/locales/es/portal.json

```diff
@@ -10,14 +10,14 @@
     "{{memberEmail}} will no longer receive emails when someone replies to your comments.": "{{memberEmail}} ya no recibirá correos electrónicos cuando alguien responde a tus comentarios.",
     "{{memberEmail}} will no longer receive this newsletter.": "{{memberEmail}} ya no recibirá este boletín.",
     "{{trialDays}} days free": "{{trialDays}} días gratis",
-    "+1 (123) 456-7890": "",
+    "+1 (123) 456-7890": "+1 (123) 456-7890",
     "A login link has been sent to your inbox. If it doesn't arrive in 3 minutes, be sure to check your spam folder.": "Se ha enviado un enlace de inicio de sesión a tu correo. Si no llega en 3 minutos, revisa tu carpeta de spam.",
     "Account": "Cuenta",
-    "Account details updated successfully": "",
+    "Account details updated successfully": "Los detalles de la cuenta se acctualizarion con éxito",
```

Comment:
There's still an extra 'c' and 'i' in actualizaron.

Reply:
Done.